Components/Cart/cart.css

```diff
@@ -71,30 +64,52 @@ body {
 
 .item-price-container-row {
     display: flex;
-    flex-flow: row wrap;
     justify-content: center;
     align-items: center;
+    margin-top: 3rem;
 }
 
-.cart-card {
+.horizontal-card {
     display: flex;
-    flex-direction: row;
-    width: 40%;
-    background-color: var(--white);
-    padding: 4rem 1rem;
-    margin: 2rem 0;
-    border-radius: 3px;
+    justify-content: center;
+    width: 40rem;
+    border-radius: 10px;
+    box-shadow: 2px 2px 8px black;
+}
+
+.horizontal-card-image-container {
+    height: auto;
+}
+
+.horizontal-card-image {
+    object-fit: cover;
+}
+
+.horizontal-card-content {
+    margin: 0;
+    padding: 0 1rem;
+}
+
+.horizontal-card-content-footer {
+    width: 100%;
+    flex-direction: column;
+}
+
+.cart-price-card-title {
+    margin-left: 1.5rem;
+
 }
 
 .cart-price-detail-card {
     display: flex;
     flex-direction: column;
     align-items: left;
     justify-content: space-between;
-    width: 35%;
-    margin: 0 2rem 4.5rem 2rem;
-    padding: 2rem 0 2rem 0;
+    width: 25rem;
+    margin: 0rem 2rem 0rem 2rem;
+    padding: 2rem 0 0rem 0;
```

Comment:
why not to use shorthand  for margin here, margin: ``` 0 2rem; ```

Reply:
Sure 👍🏻